ocs_ci/helpers/managed_services.py

```diff
@@ -350,6 +352,21 @@ def verify_storageclient_storageclass_claims(storageclient):
         log.info(f"Verified Storageclassclaim and Storageclass {sc_claim.name}")
 
 
+def verify_storageconsumers():
+    """
+    Verify that all storage consumers are in Ready state
+    """
+    sconsumer_ocp = OCP(
+        kind=constants.STORAGECONSUMER, namespace=config.ENV_DATA["cluster_namespace"]
+    )
+    sconsumers = sconsumer_ocp.get()["items"]
+    for sconsumer in sconsumers:
```

Comment:
one of the features of 4.16 is to change consumer names to client names on Client dashboard - https://issues.redhat.com/browse/RHSTOR-5500.
This test would probably fail if we don't change the names to their client names 
![image](https://github.com/red-hat-storage/ocs-ci/assets/61454420/f892df9d-bf35-4316-97ad-3a52ffcbfb1f)